Makefile

```diff
@@ -23,7 +24,7 @@ all: build
 build:
 	GOBIN=$(BINDIR) $(GO) install $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' ./
 
-# usage: make clean build-cross dist VERSION=v2.0.0-alpha.3
+# usage: make clean build-cross dist VERSION=1.0.0
```

Comment:
Will we supply the VERSION now that we're looking to git for version string?

Reply:
The git version is unrebased.  The one that's working is the one that Ryan made.  WIll followup with a PR to remove one or the other versioning infra.